### ISSUE TITLE
If subject name contains substring GRE, multi echo GRE correction fails

### ISSUE THREAD

**Issue by AlanKuurstra** [khanlab/tar2bids#11]
https://github.com/khanlab/tar2bids/blob/4cbbaf7c3edcd22c7a7b193598b9220ede15a8eb/tar2bids#L324

If the tech puts GRE as part of the patient name, then renaming fails during the correctMultiEcho script.

eg.
```
renaming: /output/sub-GRE/anat/sub-GRE_part-mag_echo_GRE1.json to /output/sub-GRE.json
renaming: /output/sub-GRE/anat/sub-GRE_part-mag_echo_GRE1.nii.gz to /output/sub-GRE.nii.gz
renaming: /output/sub-GRE/anat/sub-GRE_part-mag_echo_GRE2.json to /output/sub-GRE.json
renaming: /output/sub-GRE/anat/sub-GRE_part-mag_echo_GRE2.nii.gz to /output/sub-GRE.nii.gz
renaming: /output/sub-GRE/anat/sub-GRE_part-mag_echo_GRE3.json to /output/sub-GRE.json
renaming: /output/sub-GRE/anat/sub-GRE_part-mag_echo_GRE3.nii.gz to /output/sub-GRE.nii.gz
renaming: /output/sub-GRE/anat/sub-GRE_part-mag_echo_GRE4.json to /output/sub-GRE.json
renaming: /output/sub-GRE/anat/sub-GRE_part-mag_echo_GRE4.nii.gz to /output/sub-GRE.nii.gz
renaming: /output/sub-GRE/anat/sub-GRE_part-mag_echo_GRE5.json to /output/sub-GRE.json
renaming: /output/sub-GRE/anat/sub-GRE_part-mag_echo_GRE5.nii.gz to /output/sub-GRE.nii.gz
```

The workaround is to edit the patient name in the dicom server's web interface:

![Screenshot from 2020-01-09 10-30-28_](https://user-images.githubusercontent.com/27964245/72081818-28a7ee00-32cd-11ea-9c7e-349665d14800.png)

**Comment by akhanf** [comment]
@Park-Patrick would you be able to update the parsing in the tune-up scripts to fix this problem?

**Comment by Park-Patrick** [comment]
addressed in PR #12

**Comment by akhanf** [comment]
Looks like this was fixed so closing
